Bash_scripts/HW28/Jenkinsfile

```diff
@@ -0,0 +1,48 @@
+pipeline {
+  agent any
+  stages {
+    stage('Shell script 0') {
+      steps {
+        sh '''#!/bin/bash
+
+cd $PATHTO
+SHELL_SCRIPTS=$(find . -name \'*.sh\')
+
+for var in $SHELL_SCRIPTS ; do
+shellcheck "$var"
+done'''
+      }
+    }    
+    stage('Shell script 1') {
+      steps {
+        sh '''#!/bin/bash
+
+DOCKERFILE=$(find "$PATHTO" -name \'Dockerfile\')
+
+for var in $DOCKERFILE ; do
+dockerfile_lint  -f "$var"
+done'''
+      }
+    }
+ stage('Push images') {
+  agent any
+  steps {
+    script {
+      def dockerImage = docker.build("${REPOUSR}/${NAMEIM}", "-f ${PATHTO}/Dockerfile .")
+      docker.withRegistry('', 'dockerhub') {
+        dockerImage.push("${VERSION}_${GIT_COMMIT}_${BUILD_NUMBER}")
+      }
+      echo "Pushed Docker Image: ${NAMEIM}"
+    }
+    sh "docker rmi ${REPOUSR}/${NAMEIM}:${VERSION}_${GIT_COMMIT}_${BUILD_NUMBER} "
+  }
+}   
+
+  }
+  parameters {
+    string(name: 'PATHTO', defaultValue: 'Bash_scripts/HW27', description: null)
+    string(name: 'VERSION', defaultValue: 'v1', description: null)
+    string(name: 'NAMEIM', defaultValue: 'hw28', description: null)
+    string(name: 'REPOUSR', defaultValue: 'engelko', description: null)
```

Comment:
Better to use underscore for var name `REPO_USER` and so on

Reply:
fixed